webui/src/pages/identity/Identity.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+
+import { Link, Paper, Typography } from '@material-ui/core';
+import Avatar from '@material-ui/core/Avatar';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import InfoIcon from '@material-ui/icons/Info';
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+
+import { IdentityFragment } from '../../components/Identity/IdentityFragment.generated';
+
+import { useGetUserStatisticQuery } from './GetUserStatistic.generated';
+
+const useStyles = makeStyles((theme) => ({
+  main: {
+    maxWidth: 1000,
+    margin: 'auto',
+    marginTop: theme.spacing(3),
+  },
+  content: {
+    padding: theme.spacing(0.5, 2, 2, 2),
+    wordWrap: 'break-word',
+  },
+  large: {
+    minWidth: 200,
+    minHeight: 200,
+    margin: 'auto',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+  heading: {
+    marginTop: theme.spacing(3),
+  },
+  header: {
+    ...theme.typography.h4,
+    wordBreak: 'break-word',
+  },
+  infoIcon: {
+    verticalAlign: 'bottom',
+  },
+}));
+
+type Props = {
+  identity: IdentityFragment;
+};
+const Identity = ({ identity }: Props) => {
+  const classes = useStyles();
+  const user = identity;
+
+  const { loading, error, data } = useGetUserStatisticQuery({
+    variables: {
+      authorQuery: 'author:' + user?.humanId,
```

Comment:
as much as possible, you should make the queries with the full ID (especially if it's not visible to the user) to avoid an ID collision

Reply:
Oh yeah, my fault. Will fix.

Reply:
This should work. https://github.com/MichaelMure/git-bug/pull/605/commits/774bae6d432c13cfa0de3ddc2f111927743243fe